packages/aragon-wrapper/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@aragon/wrapper",
   "version": "5.0.0-rc.3",
   "description": "Library for Aragon client implementations",
-  "main": "dist/index.js",
+  "main": "dist/aragon-wrapper.js",
```

Comment:
What do you think about keeping an `index.js` file that just re-exports the other modules, to be more similar to other npm packages?

We also do some re-exporting in the current `index.js` that could be removed from `aragon-wrapper.js`.

Reply:
I like the idea. I also had mixed feelings about this given all the other npm packages I've seen.